packages/dd-trace/src/git_metadata.js

```diff
@@ -0,0 +1,216 @@
+
```

Comment:
Can this file be decoupled in multiple files for the metadata reader, HTTP client, etc? Also, let's move this to the CI visibility folder for now until we normalize data reporting.

Reply:
I've moved the file. As for the breaking down of the file, I'm not sure I'll be able to do much. The algorithm is very specific and can't easily be abstracted. I'll try though